utils/sane-backends/files/saned.init

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh /etc/rc.common
+
+START=90
+STOP=10
+USE_PROCD=1
+
+sanedconf() {
+	local cfg="$1"
+	config_get user "$cfg" user nobody
+	config_get portrange "$cfg" portrange
+	config_get acl "$cfg" acl
+}
+
+start_service() {
+	local user hplip portrange acl
+
+	. /lib/functions.sh
+
+	config_load saned
+	config_foreach sanedconf saned
+
+	rm -f /var/etc/saned.conf
```

Comment:
No need to remove, touch, chmod. Just create it like dnsmasq does:

echo "# auto-generated config file from /etc/config/dhcp" > $CONFIGFILE